collect_app/src/main/java/org/odk/collect/android/upload/UploadException.java

```diff
@@ -12,7 +12,7 @@
  * the License.
  */
 
-package org.odk.collect.android.upload.result;
+package org.odk.collect.android.upload;
 
 /**
  * An exception that results in the cancellation of an instance upload, and the presentation of an
```

Comment:
This is a good place to say a bit about how the various possibilities are represented: success, failure, fatal failure, retry (i.e. `UploadException` is anything non-successful, doesn't distinguish between failure and fatal failure, and there's only one kind of retryable failure, `UploadAuthRequestedException`).